### DISCUSSION TITLE
Yakuza Kiwami 60fps (TitleID: 0100C9801FEE6000)

### DISCUSSION THREAD

**Discussion by WXDFever** [StevensND/switch-port-mods#28] [Mod Request]
Yakuza Kiwami 60fps (TitleID: 0100C9801FEE6000)

**Comment by StevensND** [comment]
As far as I know: Game is having issues at 60FPS so cutscenes must be locked to 30FPS.

I haven't check if by default the game runs at 30FPS or not.

**Reply by WXDFever** [reply]
Ah, I see. thanks for letting me know
